portfolio2.css

```diff
@@ -0,0 +1,227 @@
+* {
+    box-sizing: border-box;
+	margin: 0;
+	padding:0;
+	font-family: ubuntu, sans-serif;
+}
+
+body {
+    margin: 0px;
+    /*background-image: url("https://images.pexels.com/photos/5044497/pexels-photo-5044497.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260");
+    background-color: #cccccc;*/
+
+   }
+	
+  div.links {
+		border-bottom-style: solid;
+		border-bottom-color: Lightgrey;
+		list-style: none;
+		margin: 0 2px;
+		padding: 10px;
+		display: flex;
+		justify-content: space-evenly;
+		transform: translate(0%, -1132%);
+		width:100%;
+		
+		
+	 }
+		
+	.links a{
+		display:inline;
+		justify-content: space-between;
+		color: #7393B3;
+		line-height:1.5em;
+		padding-left: 25px;
+		text-decoration:none;
+		height:90px;
+		font-size: 40px;
+		border-bottom-style: hidden;
+		
+	}
+	
+	.links a:hover{
+		border:2px #0096FF solid;
+		border-radius:5px;
+	}
+	
+	.container {
+		padding-bottom:0px;
+	}
+	
+	button{
+	    background-color: #fff;
+	    color: #7393B3;
+	    padding: 22px 27px;
+	    text-align: center;
+	    text-decoration: none;
+	    font-size: 29px;
+	    height: 11px;
+	    width: 20px;
+	    margin: 10px;
+	    border-radius: 5px;
+	}
+	
+	img{
+	    max-height: 20%;
+	    max-width: 100%;
+		margin-bottom:-116px;
+	    
+	}
+	
+	div.aboutme {
+	    border: 2px transparent solid;
+	    background-color: #53DFD1;
+	    border-top-color: black;
+	    border-bottom-color: black;
+	    background-color: #AEC6CF;
+	    padding-bottom: 200px;
+	    padding-top: 13px;
+	    width: 100%;
+	    margin: 30px 46px 10px 0px;
+	    padding-top: 21px;
+	}
+		
+	
+	
+	h2 {
+	    padding: 13px 35px;
+	}
+	
+
+	
+	div.aboutme div {
+		display:flex;
+		justify-content:center;
+		text-align:center;
+		padding-left:10px;
+	}
+	
+	div.aboutme p {
+		margin:10px 10px 10px 10px;
+		
+		
+	}
+	
+	
+	div.portfolio {
+	    display: flex;
+	    border: 2px transparent solid;
+		padding-right: 167px;
+	    background-color: #53DFD1;
+	    border-top-color: black;
+	    border-bottom-color: black;
+	    background-color: #AEC6CF;
+	    padding-bottom: 28px;
+	    padding-top: 13px;
+	    width: 100%;
+	    margin: 30px 46px 10px 0px;
+	    padding-top: 21px;
+	    place-content: space-between;
+	}
+	
+	
+	div.portfolio div {
+		display:flex;
+		justify-content:left;
+		text-align:center;
+		height:150px;
+		
+	}
+	
+	
+	div.portfolio p {
+	    color: black;
+	    border: 1px black solid;
+	    border-left-color: black;
+	    background-color: white;
+	    padding-right: 167px;
+	    padding-left: 20px;
+	    display: inline;
+	    padding-bottom: 14px;
+		
+	}
+	
+	img.one{
+		margin: 20px 20px 20px 20px;
+		max-height:100px;
+		
+	}
+	img.two{
+		margin: 20px 20px 20px 20px;
+		max-height:100px;
+	}
+	img.three{
+		margin: 20px 20px 20px 20px;
+		max-height:100px;
+	}
```

Comment:
can just apply these rules to an `img` ruleset if you apply background image to the main
could also be written as `margin: 20px` instead